src/io/flutter/run/bazel/FlutterBazelConfigurationEditorForm.java

```diff
@@ -63,4 +55,14 @@ protected void applyEditorTo(@NotNull final BazelRunConfig configuration) throws
   protected JComponent createEditor() {
     return myMainPanel;
   }
+
+  @Nullable
+  private String getTextValue(@NotNull String textFieldContents) {
+    return StringUtil.nullize(textFieldContents.trim(), true);
+  }
+
+  @Nullable
+  private String getTextValue(JTextField textField) {
```

Comment:
If we only ever use getTextValue on a JTextField, consider removing the overloaded method above.